src/main/java/it/gov/pagopa/payhub/activities/activity/classifications/IufClassificationActivityImpl.java

```diff
@@ -0,0 +1,89 @@
+package it.gov.pagopa.payhub.activities.activity.classifications;
+
+import it.gov.pagopa.payhub.activities.dao.ClassifyDao;
+import it.gov.pagopa.payhub.activities.dao.PaymentsReportingDao;
+import it.gov.pagopa.payhub.activities.dao.TreasuryDao;
+import it.gov.pagopa.payhub.activities.dto.classifications.ClassifyDTO;
+import it.gov.pagopa.payhub.activities.dto.classifications.IufClassificationActivityResult;
+import it.gov.pagopa.payhub.activities.dto.paymentsreporting.PaymentsReportingDTO;
+import it.gov.pagopa.payhub.activities.dto.treasury.TreasuryDTO;
+import it.gov.pagopa.payhub.activities.exception.PaymentsClassificationSaveException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+/**
+ * Interface for defining an activity to process payment reporting classifications based on IUF.
+ */
+@Slf4j
+@Lazy
+@Component
+public class IufClassificationActivityImpl implements IufClassificationActivity {
+    private final PaymentsReportingDao paymentsReportingDao;
+    private final TreasuryDao treasuryDao;
+    private final ClassifyDao classifyDao;
+
+    private static final String TES_NO_MATCH = "TES_NO_MATCH";
```

Comment:
let's await the other PR, in order to use the same Enum (the enum should be set as type of the field in ClassifyDTO

Reply:
done